ui/job-view/pushes/PushJobs.jsx

```diff
@@ -96,6 +109,23 @@ class PushJobs extends React.Component {
     jobInstance.toggleRunnableSelected();
   };
 
+  handleGroupClick = (ev, groupNode, groupInstance) => {
+    if (!groupInstance.state.expanded || !ev.metaKey) {
+      return;
+    }
+
+    const { toggleSelectedRunnableJob } = this.props;
+
+    for (const jobButton of groupNode.querySelectorAll('.runnable-job-btn')) {
```

Comment:
I uses `querySelectorAll` here because:
iterating over `groupInstance.props.jobs` can get `job.signature`, but I cannot find a way to get `jobInstance` from it, or perform `jobInstance.toggleRunnableSelected()` or equivalent.